ui/component/commentsList/index.js

```diff
@@ -30,7 +30,7 @@ const select = (state, props) => {
 
   if (topLevelComments.length > 0) {
     topLevelComments.map(
-      (comment) => Boolean(makeSelectClaimForUri(comment.channel_url)(state)) && resolvedComments.push(comment)
+      (comment) => makeSelectClaimForUri(comment.channel_url)(state) !== undefined && resolvedComments.push(comment)
```

Comment:
I'm think using `map` for the purpose of iterating is not efficient, since `map` would create the mapped array internally, whether assigned or not.

`filter` makes more sense and readable, as it conveys the intention directly.

`resolvedComments = topLevelComments.filter(.... !== undefined)`